src/amo/components/RatingsByStar/index.js

```diff
@@ -0,0 +1,149 @@
+/* @flow */
+import makeClassName from 'classnames';
+import invariant from 'invariant';
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+
+import { fetchGroupedRatings } from 'amo/actions/reviews';
+import { withFixedErrorHandler } from 'core/errorHandler';
+import translate from 'core/i18n/translate';
+import LoadingText from 'ui/components/LoadingText';
+import Icon from 'ui/components/Icon';
+import type { GroupedRatingsType } from 'amo/api/reviews';
+import type { AppState } from 'amo/store';
+import type { AddonType } from 'core/types/addons';
+import type { ErrorHandlerType } from 'core/errorHandler';
+import type { I18nType } from 'core/types/i18n';
+import type { DispatchFunc } from 'core/types/redux';
+
+import './styles.scss';
+
+type Props = {|
+  addon: AddonType | null,
+|};
+
+type InternalProps = {|
+  ...Props,
+  dispatch: DispatchFunc,
+  errorHandler: ErrorHandlerType,
+  groupedRatings?: GroupedRatingsType,
+  i18n: I18nType,
+|};
+
+export class RatingsByStarBase extends React.Component<InternalProps> {
+  componentWillMount() {
+    // TODO: this is intended to load on the server (before mount) but it
+    // does not.
+    // See: https://github.com/mozilla/addons-frontend/issues/5854
```

Comment:
I guess this might get potentially fixed with the updates @willdurand referenced earlier (in an email)?

Reply:
Yes, it will. We're targeting that fix for Q4 so I wanted to make sure and leave this as `componentWillMount()` to make it clear what the intention was.